readthedocs/api/v3/permissions.py

```diff
@@ -16,6 +16,7 @@ def has_permission(self, request, view):
         if is_authenticated:
             if view.basename == 'projects' and any([
                     view.action == 'list',
+                    view.action == 'create',  # used to create Form in BrowsableAPIRenderer
```

Comment:
Do we want to support that? I feel like creating a project via the web form is probably not going to be the best UX.

Reply:
I've been thinking on making the browsable API consistent with what the API offers. Besides consistency, it helps to explore it and have a better understanding of what's available and what can be done: not "self-documented" but in that direction.

Reply:
On the other hand, removing the `create` from this check it fails with 404 when accessing to `/api/v3/projects/` because of an internal DRF check. Actually, that's how I realized that I needed it :)